src/@types/index.d.ts

```diff
@@ -5,3 +5,37 @@ interface Client {
 interface Fastly {
   client: Client;
 }
+
+interface Host {
+  host: string;
+  lookup: string;
+}
+
+interface Settings {
+  initial_delay: number;
+  max_tasks: number;
+  report_errors: boolean;
+  sample_rate: number;
+}
+
+interface Server {
+  datacenter: string;
+}
+
+interface Task {
+  name: string;
+  req_header: string;
+  resource: string;
+  resp_header: string;
+  type: string;
+  weight: number;
+}
+
+interface Config {
+  apiKey: string;
```

Comment:
Apologies, I've dropped `apiKey` from the top-level since we last spoke, so no need to include this. It now lives at `settings.token` as per above comment.